loopbackStarter/README.md

```diff
@@ -12,6 +10,23 @@ $ npm install -g strongloop
 If you run into any issues, see [Installing StrongLoop](http://docs.strongloop.com/display/LB/Installing+StrongLoop)
 for more information.
 
+You must have already installed the [Cloud Foundry Command Line Interface (cf CLI)](https://docs.cloudfoundry.org/cf-cli/) for deploying the app to Bluemix.
+
+## Deploying to Bluemix
+
+**Prerequisites**
+
+This app is pre-configured to be deployed to Bluemix, however, the following prerequisites must be met before it can be run successfully.
+
+- Modify `manifest.yml` and `package.json` to replace "loopback-starter-app" with a unique name for your app instance.
+- Push the app to Bluemix using `cf push <name of your app>`.
+- Create a [MongoDB](https://console.ng.bluemix.net/catalog/services/compose-for-mongodb) named `loopback-mongodb-starter`, and bind the service to your app instance.
```

Comment:
this brings up another question of if we want to use a "non-freemium" database service for a boilerplate... the compose services all incur a cost:
```
Creating service instance loopback-mongodb-starter in org svennam@us.ibm.com / space dev as svennam@us.ibm.com...
OK

Attention: The plan `Standard` of service `compose-for-mongodb` is not free.  The instance `loopback-mongodb-starter` will incur a cost.  Contact your administrator if you think this is in error.